src/api/ApiClient.ts

```diff
@@ -3,12 +3,22 @@
 import { PlaygroundClient } from '@wp-playground/client';
 import { Post } from '@/api/Post';
 import { Settings } from '@/api/Settings';
+import { User } from '@/api/User';
 import { PostContent, PostDate, PostTitle } from '@/parser/post';
 
 export interface CreatePostBody {
 	guid: string;
 }
 
+export interface CreateUserBody {
+	username: string;
+	email: string;
+	password: string;
+	roles?: [ string ]; // default roles: administrator, editor, author, subscriber (default)
```

Comment:
Should we make this a `string` instead of `[ string ]`?  As we discussed, it's unexpected behaviour to have a user with multiple roles, so making it a `string` would prevent it.

Reply:
I kept it as an array as per docs. We shouldn't go against that IMO. Maybe the UI hasn't caught up but technically multiple roles are supported now or will be in future?

Reply:
The way I see `ApiClient` is that, instead of being a generic WordPress REST API client, it would be optimized for the needs of our app. Part of its purpose is to abstract the complexities of the WP REST API from the rest of the app.

This would mean that the types defined in `ApiClient` would not be directly mimic the types of the WP REST API, but instead are defined in way that better fits the model of our app.

In this case, the WP REST API makes it possible to assign many roles to a user, but in the domain of our app, we might decide that a user can only have one role. If instead we decide that the user can have multiple roles, we then need to expose that fact in the UI so that the user of our app can assign multiple roles to a user.

I think for now we could make it so that a user can only have a single role, which will simplify things for us. If we make that decision, we can still re-evaluate In the future, and in that case, we would modify the `ApiClient`, to instead expose a `roles` property, or have both `role` and `roles` properties.

Reply:
> This would mean that the types defined in ApiClient would not directly mimic the types of the WP REST API, but instead are defined in way that better fits the model of our app.

Sold!

Reply:
@psrpinto Take a look - https://github.com/WordPress/try-wordpress/pull/55/commits/269ea81022cd2ea124c5b1447fa01881669eb448

Reply:
Perfect!